BuildScripts/LocalBuildSettings-template.txt

```diff
@@ -0,0 +1,12 @@
+{
+	"AndroidProjectPath": "../Mirage-Android-WebView-Plugin",
+	"UnityProjectPath": "../Mirage-Widget",
+	"UnityPath": "C:/Program Files/Unity/Hub/Editor/2021.3.8f1/Editor/Unity.exe",
+	"KeystorePath": "C:/repos/Fuzzball-Fighters/FF.keystore",
```

Comment:
no reason to commit your local settings

Reply:
Make them independent of your environment or add to ignore

Reply:
@Vukiz They are already added to ignore. 
LocalBuildSettings.txt is used by a script and is ignored, LocalBuildSettings-template.txt is just an example not used by any script. The user of this repo is supposed to copy this file, rename it to LocalBuildSettings.txt and set up his own local paths. I will make a doc for this.